components/clarinet-files/src/network_manifest.rs

```diff
@@ -893,6 +896,16 @@ impl NetworkManifest {
                         }
                     },
                 ),
+                stacks_signer_image_url: devnet_config
+                    .stacks_signer_image_url
+                    .take()
+                    .unwrap_or_else(|| {
+                        if devnet_config.use_nakamoto.unwrap_or(false) {
+                            DEFAULT_STACKS_NODE_IMAGE_NAKA.to_string()
```

Comment:
Currently this won't work with the default naka image, right? Do we need to update this const to be beta2?

Reply:
Done! Up to beta4